chain/epoch-manager/src/lib.rs

```diff
@@ -1013,19 +1017,23 @@ impl EpochManager {
 
         let next_epoch_id = self.get_next_epoch_id(last_block_hash)?;
         let epoch_id = self.get_epoch_id(last_block_hash)?;
-        debug!(target: "epoch_manager",
-            "epoch id: {:?}, prev_epoch_id: {:?}, prev_prev_epoch_id: {:?}",
-            next_next_epoch_id, next_epoch_id, epoch_id
+        tracing::debug!(target: "epoch_manager",
+            ?next_next_epoch_id,
+            ?next_epoch_id,
+            ?epoch_id,
+            "epoch ids"
```

Comment:
Could you please adjust names to align with previous version and remove message?